pastas/model.py

```diff
@@ -1367,22 +1385,25 @@ def fit_report(self, output="full"):
             line=string.format("", fill='=', align='>', width=width),
             parameters=parameters)
 
-        # w = []
-        #
-        # warnings = "Warnings\n{line}\n".format(
-        #     "",
-        #     line=string.format("", fill='=', align='>', width=width)
-        # )
-        #
-        # for n, warn in enumerate(w, start=1):
-        #     warnings = warnings + "[{}] {}\n".format(n, warn)
-        #
-        # if len(w) == 0:
-        #     warnings = ""
-
-        report = "{header}{basic}{parameters}".format(
-            header=header, basic=basic, parameters=parameters
-        )
+        if output == "full":
+            cor = dict()
+            pcor = self.fit.pcor
+            for idx in pcor:
+                for col in pcor:
+                    if (np.abs(pcor.loc[idx, col]) > 0.5) and (idx != col) \
```

Comment:
I set the value here to 0.5 (was 0.3), as stated in the description in fit_report() later on. Was 0.5 or 0.3 the intended value?

Reply:
Good point! 0.5 was the intended value. I think we can work on the representation of these values later and make it an input argument.